File: daily/python/2076_buy_two_choco.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        min1 = min2 = float('inf')
        for p in prices:
            if p < min1:
                min2 = min1
                min1 = p
            elif p < min2:
                min2 = p
        if min1 + min2 > money:
            return money
        return money - min1 - min2

File: daily/rust/2076_buy_two_choco.rs
impl Solution {
    pub fn by_two_choco(prices: Vec<i32>, money: i32) -> i32 {
        let (mut min1, mut min2) = (std::i32::MAX, std::i32::MAX);
        for p in prices {
            if p < min1 {
                min2 = min1;
                min1 = p;
            } else if p < min2 {
                min2 = p;
            }
        }
        if min1 + min2 > money {
            return money;
        }
        money - min1 - min2
    }
}
